tests/lava/lib/optimization/utils/test_solver_benchmarker.py

```diff
@@ -0,0 +1,70 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+# See: https://spdx.org/licenses/
+

```

Comment:
I would appreciate if you could add an issue, claiming that the unit tests for the solver_benchmarker need extension once the release stress is over.